packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -194,13 +197,18 @@ export const GridComponent = React.forwardRef<HTMLDivElement, GridComponentProps
                     ref={columnsContainerRef}
                     height={internalOptions.headerHeight}
                   >
+                    <ScrollArea scrollDirection="left" apiRef={apiRef} />
```

Comment:
this needs to be abstracted inside columnsHeader maybe?

Reply:
Unfortunately, it has to be outside of the `ColumnsHeader` component because of the width of the `ColumnsHeader` (it overflows outside of the visible area of the `ColumnsContainer`). If I put it there then the right scroll area will be not be positioned correctly.

What I can do is move it inside the `ColumnsContainer`. That way it will still be abstracted, what do you think @dtassone ?

Reply:
Done